src/engine/controls/cuecontrol.h

```diff
@@ -203,6 +203,7 @@ class CueControl : public EngineControl {
     ControlObject* m_pQuantizeEnabled;
     ControlObject* m_pClosestBeat;
     bool m_bypassCueSetByPlay;
+    ControlValueAtomic<double> m_oldSeekOnLoadPosition;
```

Comment:
Why "m_oldSeekOnLoadPosition" instead of just "m_seekOnLoadPosition"? Otherwise I would expect that there also is a *current* or *new* position.

Reply:
It is like that. "m_oldSeekOnLoadPosition" is the position that has been taken because of the old cue points before the analysis has finished. The new position is calculated on the fly and copied into old. "m_seekOnLoadPosition" does not work for me for the case. Do you have a better fitting name?

Reply:
You explained that m_oldSeekOnLoadPosition always reflects the current *seek-on-load* position that is valid at that point in time. After the analysis has finished it is replaced if not already set to a valid position. So why old? It does never get *old* since it is updated.

Reply:
The only propose is to compare the old position with the new one. In such cases I use always the term old. 
This way it is clear "don't use the value again, it is old".